water-sanitation-and-hygiene-for-development.csl

```diff
@@ -0,0 +1,618 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Water Sanitation and Hygiene for Development</title>
+    <title-short>WASHEV</title-short>
+    <id>http://www.zotero.org/styles/water-sanitation-and-hygiene-for-development</id>
+    <link href="http://www.zotero.org/styles/water-sanitation-and-hygiene-for-development" rel="self"/>
+    <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
```

Comment:
Also, add a template link to the style you based this on (apa I take it?) i.e.
`<link href="http://www.zotero.org/styles/apa" rel="template"/>`